inline_verifier.go

```diff
@@ -229,20 +230,49 @@ type InlineVerifier struct {
 	sourceStmtCache *StmtCache
 	targetStmtCache *StmtCache
 	logger          *logrus.Entry
+
+	// Used only for the ControlServer initiated VerifyDuringCutover
+	backgroundVerificationResultAndStatus VerificationResultAndStatus
+	backgroundVerificationErr             error
+	backgroundVerificationWg              *sync.WaitGroup
 }
 
+// This is called from the control server, which is triggered by pushing Run
+// Verification during cutover.
+// This step is necessary to ensure the binlogs are verified in Ghostferry.
 func (v *InlineVerifier) StartInBackground() error {
-	// not needed?
+	if v.verifyDuringCutoverStarted.Get() {
+		return errors.New("verification during cutover has already been started")
+	}
+
```

Comment:
Also looks like we may be missing some other checks that exist in the `IterativeVerifier`?

https://github.com/Shopify/ghostferry/blob/3113d9c4da8ac6e01b7ff75c9e852f11dcd58584/iterative_verifier.go#L245-L251

Reply:
Done.